pkg/overlay/overlay.go

```diff
@@ -146,6 +151,12 @@ func mountWithMountProgram(mountProgram, overlayOptions, mergeDir string) error
 	return nil
 }
 
+// mountNatively mounts an overlay at mergeDir using the kernel's mount()
+// system call.
+func mountNatively(overlayOptions, mergeDir string) error {
+	return mount.Mount("overlay", mergeDir, "overlay", overlayOptions)
+}
+
```

Comment:
@nalind, I think this is OK, in fact I prefer the func to be used prior to it being called, but it is in a different order than your 1.31 commit, https://github.com/containers/buildah/pull/5965/commits/18f1db77f344636fdd8e5eedc3b9fd9e5bb89eee.  Any concerns?

Reply:
I can move mine.  It's better that future backports are kept as simple as possible.